MANIFEST.in

```diff
@@ -11,19 +11,14 @@ include *.md
 include *.html
 include *.sh
 include pytest.ini
-prune docs/_build
-recursive-include reana *.py
-recursive-include docs *.py
-recursive-include docs *.png
-recursive-include docs *.rst
-recursive-include docs *.txt
 recursive-include tests *.py
 recursive-include helm *.yaml
 recursive-include helm *.txt
 recursive-include helm *.helmignore
 recursive-include helm *.lock
 recursive-include benchmark *.py
 recursive-include scripts *.sh
+recursive-include images *.png
```

Comment:
There is one warning:

```console
$ check-manifest
lists of files in version control and sdist do not match!
missing from VCS:
  helm/reana/Chart.lock
```

We can configure ignore patterns to silence this warning.  (We don't want to add the file.)

Can you edit `setup.cfg` to put ignore statement for `check-manifest` for this file?